apps/aecore/test/aec_chain_tests.erl

```diff
@@ -1274,30 +966,17 @@ longest_block_chain_test_() ->
 
                %% Check top block changed.
                ?assertEqual({ok, HA3}, aec_chain:top_header()),
-               ?assertEqual({ok, BA2}, aec_chain:top_block()),
-
+	       
+	       %% !!!! Race condition here !!!!
+	       timer:sleep(2000),
```

Comment:
Isn't there any way to achieve a sync point here without resorting to timers? A TODO perhaps?

Reply:
This timer is not needed with the new implementation which only uses one server.
will remove.

Reply:
The race is not there anymore. It was an artifact in the old version. I remove this.